docs/references/javascript/user/user.mdx

```diff
@@ -160,7 +160,7 @@ Retrieves all of the organizations the user is a member of.
 
 | Type | Description |
 | --- | --- |
-| <code>Promise\<[PaginatedResponse](/docs/references/javascript/types/clerk-paginated-response)\<OrganizationMembershipResource\>\></code> | The list of organizations the user is a member of. |
+| <code>Promise\<[PaginatedResponse](/docs/references/javascript/types/clerk-paginated-response)\<OrganizationMembershipResource\>\></code> | The list of organizations the user is a member of or rejects with `ClerkAPIResponseError` or Network error. |
```

Comment:
I am not 100% sure we should add this. If you think we should not refer that this function can also throw an error we should revert this change. This change is also applied to 5 other methods (above).
cc: @alexisintech , @LekoArts

Reply:
I'll put it into the page of the ClerkPaginatedResponse, then we don't have to repeat ourselves (it's a good addition)